examples/oauth2/config.json

```diff
@@ -0,0 +1,126 @@
+{
```

Comment:
this is here as an example config to show the data that is necessary, the example would work with any config.json downloaded from the admin portal as long as they are using oauth authentication mode AND they have filled in the login url/Authorization endpoint in the Integration screen.